SKYKit.podspec

```diff
@@ -12,8 +12,6 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '7.0'
 
-  s.default_subspec = 'Core'
```

Comment:
> What will happen without default subspec?

For this question, if not specified, then all subspec would be included.
https://guides.cocoapods.org/syntax/podspec.html#default_subspecs